### PR TITLE
Include org quota in show-cf-memory-usage

### PR DESCRIPTION
**Pull request by alext**
## What

So that we can easily see the quotas assigned to each org at a glance.

## How to review

Run the task against an environment (eg `make dev show-cf-memory-usage`) and check the output looks sensible.

## Who can review

Anyone but myself.

### CODE REVIEW

**Comment by mtekel** [comment]
```
Org 'admin': quota: 2048 MB (2 GB), reserved by apps: 940 MB (0 GB)
Org 'testers': quota: 2048 MB (2 GB), reserved by apps: 256 MB (0 GB)
Org 'CATS-persistent-org': quota: 10240 MB (10 GB), reserved by apps: 256 MB (0 GB)
Org 'michael-test': quota: 2048 MB (2 GB), reserved by apps: 228 MB (0 GB)
Org 'test-org': quota: 2048 MB (2 GB), reserved by apps: 0 MB (0 GB)
Org 'heavy-testers': quota: 61440 MB (60 GB), reserved by apps: 11520 MB (11 GB)
```
I prefer MB->GB conversion was a bit more realistic. 940MB is almost 1 GB...

**Comment by alext** [comment]
> I'd prefer MB->GB conversion was a bit more realistic. 940MB is almost 1 GB...

true, but i've not changed that formatting in this PR.

File: scripts/show-cf-memory-usage.rb [hunk 1]
```diff
@@ -35,23 +39,24 @@
...
   allocated_routes += quota_def['entity']['total_routes'].to_i
+
   org_id = org['metadata']['guid']
-  org_apps_reserved_memory = JSON.load(`cf curl /v2/organizations/#{org_id}/memory_usage`)['memory_usage_in_mb']
-  apps_reserved_memory += org_apps_reserved_memory.to_i
-  puts "Memory reserved by apps in org '#{org['entity']['name']}': #{org_apps_reserved_memory} MB"
+  org_apps_reserved_memory = JSON.load(`cf curl /v2/organizations/#{org_id}/memory_usage`)['memory_usage_in_mb'].to_i
+  apps_reserved_memory += org_apps_reserved_memory
+
+  puts "Org '#{org['entity']['name']}': quota: #{format_memory(org_memory_quota)}, reserved by apps: #{format_memory(org_apps_reserved_memory)}"
```
Comment (saliceti) [inline scripts/show-cf-memory-usage.rb]: It works. I'm getting:

```
Org 'admin': quota: 10240 MB (10 GB), reserved by apps: 4068 MB (3 GB)
```

I wonder if it reads more naturally to represent a fraction, such as:

```
Org 'admin': 4068 MB (3 GB) reserved by apps / 10240 MB (10 GB) quota
```

Comment (alext) [inline scripts/show-cf-memory-usage.rb]: Fair point. I've updated it to make this change.

**Comment by alext** [comment]
I've also added a commit to improve the rounding of smaller memory amounts, which should address @mtekel's comment.

**Comment by mtekel** [comment]
```
Org 'admin': 940 MB (0.9 GB) reserved by apps / 2048 MB (2 GB) quota
Org 'testers': 256 MB (0.3 GB) reserved by apps / 2048 MB (2 GB) quota
Org 'CATS-persistent-org': 256 MB (0.3 GB) reserved by apps / 10240 MB (10 GB) quota
Org 'michael-test': 228 MB (0.2 GB) reserved by apps / 2048 MB (2 GB) quota
Org 'test-org': 0 MB (0 GB) reserved by apps / 2048 MB (2 GB) quota
Org 'heavy-testers': 11520 MB (11.3 GB) reserved by apps / 61440 MB (60 GB) quota
./scripts/show-cf-memory-usage.rb:43:in `block in <main>': undefined method `[]' for nil:NilClass (NoMethodError)
	from ./scripts/show-cf-memory-usage.rb:39:in `each'
	from ./scripts/show-cf-memory-usage.rb:39:in `<main>'
make: *** [show-cf-memory-usage] Error 1
```

**Comment by alext** [comment]
@mtekel and I dug into this issue, and it turns out that the issue is caused by us not fetching all pages when fetching the quotas (@mtekel's dev environment had lots of left over CATS quotas.), so the script wasn't finding the quota for the org.

This is not something related to this change, so I'll fix it in a separate PR (we're not paginating the orgs either).

**Comment by mtekel** [comment]
LG, merging

**Pull request merged**